graphql/schema/types/distro.graphql

```diff
@@ -138,6 +138,7 @@ input DistroInput {
   note: String!
   plannerSettings: PlannerSettingsInput!
   provider: Provider!
+  providerAccount: String!
```

Comment:
@Kimchelly Is there any opposition to forcing the user to client to always populate this field e.g. passing in an empty string when its first set